app/models/post.rb

```diff
@@ -3,7 +3,10 @@ class Post < ActiveRecord::Base
   has_many :comments, dependent: :destroy
   has_many :likes, dependent: :destroy
 
-  after_create :update_post_counter # Cambiado a after_create
+  validates :title, presence: true, length: { maximum: 250 }
+  validates :text, presence: true, length: { maximum: 5000 }
+
```

Comment:
- [ ] Great job so far. please as stated in the guidlines and for your app to work properly, kindly add validations a requested.
- please ensure the CommentsCounter shouldbe an integer greater than or equal to zero.
- Also, the LikesCounter should be an integer greater than or equal to zero.

Reply:
Thank you!! 🙌🙌